### PR TITLE
httpjson: support configurable response_timeout

### PR DESCRIPTION
**Pull request by gam-phon**
### Required for all PRs:

- [X] CHANGELOG.md updated (we recommend not updating this until the PR has been approved by a maintainer)
- [X] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)
- [X] README.md updated (if adding a new plugin)

### CODE REVIEW

**Comment by gam-phon** [comment]
Fixed #1650

**Comment by gam-phon** [comment]
I have a data race. I will try to fix it. I am new to go ^_^

File: plugins/inputs/httpjson/httpjson.go [hunk 1]
```diff
@@ -221,6 +225,10 @@ func (h *HttpJson) gatherServer(
 //     string: body of the response
 //     error : Any error that may have occurred
 func (h *HttpJson) sendRequest(serverURL string) (string, float64, error) {
+	// Set default values
+	if h.ResponseTimeout.Duration < time.Second {
```
Comment (sparrc) [inline plugins/inputs/httpjson/httpjson.go]: this should be set in the `init()` function instead

**Comment by gam-phon** [comment]
@sparrc I have moved the default value of ResponseTimeout to init. Thanks ^_^

**Comment by sparrc** [comment]
thanks @gam-phon

**Comment by gam-phon** [comment]
@sparrc Welcome. Thanks for merging.

**Pull request merged**